src/eShopOnBlazor/eShopOnBlazor.csproj

```diff
@@ -37,10 +37,17 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Autofac" Version="6.5.0" />
-    <PackageReference Include="EntityFramework" Version="6.4.4" />
-    <PackageReference Include="log4net" Version="2.0.15" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Log4Net.AspNetCore" Version="6.1.0" />
+    <PackageReference Include="EntityFramework" />
```

Comment:
I don't recognize this pattern for specifying default package versions. Are we sure this works?

Reply:
@danroth27 earlier the repo had this below error 
![image](https://github.com/dotnet-architecture/eShopOnBlazor/assets/136682990/07f51920-6a85-4d51-92d5-d99af72e24ae)

Resolved that error by adding PackageVersion.

Tested. Its working fine with that